CMSgov/design-system
# [WNMGDS-874] Updates to tooltip design

## Summary
- Updated the styling of the tooltip based on direction from @mliupaeter 
- Removed the inverse option for the tooltip as styling of the tooltip is going to be the same no matter what the background is 
- Updated test to remove inverse test

### Added
- Added styles and variables to allow for brand theming by Healthcare and Medicare 


## How to test or demo
- Check out the live URL here 
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-874-update-tooltip-design/components/tooltip/

- Ensure the design matches the Sketch file https://www.sketch.com/s/9d0a4d12-4820-4cd7-984b-44080a4ad4ab
